templates/todo/projects/csharp-sql-swa-func/.vscode/launch.json

```diff
@@ -0,0 +1,39 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Debug Web",
+            "request": "launch",
+            "type": "msedge",
+            "webRoot": "${workspaceFolder}/src/web/src",
+            "url": "http://localhost:3000",
+            "sourceMapPathOverrides": {
+                "webpack:///src/*": "${webRoot}/*"
+            },
+        },
+        {
+            "name": "Debug API",
+            "type": "coreclr",
+            "request": "launch",
```

Comment:
Does this require a build step as a prereq, or does this build automatically?

Reply:
func will do the build automatically.